public/app/percona/pmm-dump/components/ExportDataset/ExportDataset.tsx

```diff
@@ -0,0 +1,225 @@
+import { useAppDispatch } from 'app/store/store';
+import { useSelector } from 'app/types';
+
+import { SwitchRow } from '../../../settings/components/Advanced/SwitchRow';
+import { PMMDumpService } from '../../PMMDump.service';
+
+import { GET_SERVICES_CANCEL_TOKEN, DUMP_URL } from './ExportDataset.constants';
+import { Messages } from './ExportDataset.messages';
+import { getStyles } from './ExportDataset.styles';
+import { ExportDatasetProps } from './ExportDataset.types';
+
+const { Form } = withTypes<ExportDatasetProps>();
+
+const ExportDataset: FC<GrafanaRouteComponentProps<{ type: string; id: string }>> = ({ match }) => {
+  const styles = useStyles2(getStyles);
+  const dispatch = useAppDispatch();
+  const [selectedTimerange, setSelectedTimerange] = useState<TimeRange>();
+  const { isLoading, services: fetchedServices } = useSelector(getServices);
+
+  const flattenServices = useMemo(
+    () =>
+      fetchedServices.map((data: { params: { serviceName: string } }): SelectableValue<string> => {
+        return {
+          label: data.params.serviceName,
+          value: data.params.serviceName,
+        };
+      }),
+    [fetchedServices]
+  );
+
+  const [generateToken] = useCancelToken();
+  const [endDate, setEndDate] = useState<DateTime>(dateTime(new Date().setSeconds(0, 0)));
+  let defaultEndDate = new Date();
+  defaultEndDate.setSeconds(0, 0);
+  const [date, setDate] = useState<DateTime>(dateTime(new Date(defaultEndDate.getTime() - 6 * 1000 * 60 * 60)));
+
+  const loadData = useCallback(async () => {
+    try {
+      await dispatch(fetchServicesAction({ token: generateToken(GET_SERVICES_CANCEL_TOKEN) }));
+      await dispatch(fetchActiveServiceTypesAction());
+    } catch (e) {
+      if (isApiCancelError(e)) {
+        return;
+      }
+      console.log(e);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
+    loadData();
+    setSelectedTimerange({
+      from: dateTime(new Date('2023-10-03T04:35:46.795Z')),
+      to: dateTime(new Date('2023-10-03T10:35:46.795Z')),
+      raw: {
+        from: 'now-6h',
+        to: 'now',
+      },
+    });
+
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const history = useHistory();
+  const handleGoBack = () => {
+    history.push(DUMP_URL);
+  };
+
+  const handleSubmit = async (data: ExportDatasetProps) => {
+    console.log(data.service);
+    let serviceList;
+
+    if (date > endDate) {
+      appEvents.emit(AppEvents.alertError, ['Please select a valid time range']);
+      return;
+    }
+    if (!data?.service) {
+      serviceList = flattenServices?.map(({ value }): string | undefined => value);
+    } else {
+      console.log(data.service);
+      serviceList = data?.service?.map(({ value }): string => value);
+    }
+
+    await PMMDumpService.triggerDump(
+      serviceList,
+      date.toISOString(),
+      endDate.toISOString(),
+      data.QAN ? true : false,
```

Comment:
What's the reason behind the condition check? If you want return boolean value you can use double negation (`!!`) to convert truthy/falsy value to a boolean one:
`!!data.QAN`.